platform/stm32f0xx/scripts/gdb_flash

```diff
@@ -0,0 +1,11 @@
+set pagination off
```

Comment:
Optional: Any way to document/comment this? I know what it does but most of our other files at least have some minimal comments explaining what they do for new members.